astropy/convolution/tests/test_kernel_class.py

```diff
@@ -292,7 +292,7 @@ def test_custom_1D_kernel_list(self):
         Check if CustomKernel works with lists.
         """
         custom = CustomKernel([1, 1, 1, 1, 1])
-        assert custom.is_bool == True
```

Comment:
Yes, you're right. Here I just assumed blindly that is_bool should be bool (or maybe None).